docs/modules/ROOT/pages/system-properties.adoc

```diff
@@ -569,6 +569,14 @@ idle CPU usage but may result in decreased latency. Higher values will increase
 |int
 |Some JMX MBeans are cached  to reduce the overhead of calculating statistics. This parameter determines for how long the MBeans can go stale.
 
+|`hazelcast.journal.cleanup.threshold`
+| 1.0f
+| float
+| The cleanup threshold for the event journal, expressed as a fraction of its capacity (a value between 0.0f and 1.0f).
+When the remaining capacity falls below this threshold, a cleanup may be triggered during put operations to remove expired events (only if a TTL is configured).
+A value of 0.0f means cleanup occurs when the journal is full, while a value of 1.0f means cleanup is triggered after each put operation.
+Applies to event journals of both IMap and ICache.
```

Comment:
```suggestion
```

It applies to all data structures having journal (currently IMap and ICache) and the design is generic and with the idea that other structures would behave in the same way. So I would remove this explicit enumeration.